From 25e792c31a35b2cbf0ec771b3e9aa066cc1ab723 Mon Sep 17 00:00:00 2001
From: yipengya <yipeng.yao@intel.com>
Date: Fri, 21 Jun 2019 10:53:33 +0800
Subject: [PATCH] APL NUC: remove CPMS

CPMS is IVI car power usage, however it blocks system mobile power usage
such as auto sleep.
remove CPMS as temp walk around

Change-Id: I9bb57ac873e278e2425a57d846c51db22a0849c2
Tracked-On:
Signed-off-by: yipengya <yipeng.yao@intel.com>
---
 service/src/com/android/car/ICarImpl.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/service/src/com/android/car/ICarImpl.java b/service/src/com/android/car/ICarImpl.java
index 63ea2a74..998d0fb8 100644
--- a/service/src/com/android/car/ICarImpl.java
+++ b/service/src/com/android/car/ICarImpl.java
@@ -152,7 +152,7 @@ public class ICarImpl extends ICar.Stub {
         // Be careful with order. Service depending on other service should be inited later.
         List<CarServiceBase> allServices = new ArrayList<>();
         allServices.add(mSystemActivityMonitoringService);
-        allServices.add(mCarPowerManagementService);
+        //allServices.add(mCarPowerManagementService);
         allServices.add(mCarPropertyService);
         allServices.add(mCarDrivingStateService);
         allServices.add(mCarUXRestrictionsService);
-- 
2.22.0

